oauthenticator/traitlets.py

```diff
@@ -0,0 +1,16 @@
+from traitlets import TraitType
```

Comment:
We depend on JupyterHub, so you might be able to just import https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/traitlets.py#L93

Reply:
Ah!  I didn't notice that, thanks.

The requirements.txt allows jupyterhub 0.5, which doesn't include Callable -- Callable first shows up in 0.9.0.  Is it okay to stop supporting jupyterhub < 0.9.0?

Reply:
Oh, that's interesting. I *think* we passed that bridge a long time ago...

Regardless, let's leave it as is and pick that up in a separate PR.